sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/storage/GcsFileSystem.java

```diff
@@ -76,6 +88,38 @@ protected void copy(List<GcsResourceId> srcResourceIds, List<GcsResourceId> dest
     options.getGcsUtil().copy(toFilenames(srcResourceIds), toFilenames(destResourceIds));
   }
 
+  @VisibleForTesting
+  List<MatchResult> matchNonGlobs(List<GcsPath> gcsPaths) throws IOException {
```

Comment:
`matchSinglePaths`?

I dislike referring to the method by what it doesn't accept.